fms2_io/include/get_global_attribute.inc

```diff
@@ -32,8 +32,15 @@ subroutine get_global_attribute_0d(fileobj, &
                                              !! The broadcast will be done
                                              !! by default.
     integer :: err
+    character(len=1024), dimension(1) :: charbuf !< 1D Character buffer
+
     if (fileobj%is_root) then
         select type(attribute_value)
+            type is (character(len=*))
+                err = nf90_get_att(fileobj%ncid, &
+                                   nf90_global, &
+                                   trim(attribute_name), &
+                                   charbuf(1))
```

Comment:
Will this always work with all compilers?  Do you need to pass in the length of the string with the string?

Reply:
PGI support is going to be added in a seperate PR. This works for all **normal** compilers.

Reply:
I will be adding the work around for PGI in a separate PR. 
@colingladueNOAA Therefore, there is no need to add the test in the configure we talked about in the meeting.
I think this is ready to go in.